npm/ng-packs/packages/theme-shared/src/lib/components/internet-connection-status/internet-connection-status.scss

```diff
@@ -0,0 +1,25 @@
+.Blink {
+  animation: blinker 0.9s cubic-bezier(.5, 0, 1, 1) infinite alternate;  
+}
+@keyframes blinker {  
+  0% { color:#c1c1c1 }
+  70% { color: #DC3545 }
+  100% { color: #DC3545 }
+}
+
+.text-blinking{
+  font-size:1.2rem;
+}
+
+.statusIcon{

```

Comment:
use kebab case for css classes like
`.status-icon` also don't forget the update html